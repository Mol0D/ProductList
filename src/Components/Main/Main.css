.main{
    text-align: center;
}

.main-logo {
    animation: Main-logo-spin infinite 20s linear;
    height: 300px;
    pointer-events: none;
}

@keyframes Main-logo-spin {
    from {
        transform: rotate(0deg);
    }
    to {
        transform: rotate(360deg);
    }
}