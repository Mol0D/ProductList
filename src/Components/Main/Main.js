import React from 'react';
import './Main.css';
import logo from '../../logo.svg';

const Main = () =>{
    return (
        <div className='main'>
            <img src={logo} className="main-logo" alt="logo"/>
        </div>
    )
}

export default Main;