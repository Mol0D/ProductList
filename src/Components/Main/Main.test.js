import React from 'react';
import {shallow} from 'enzyme';
import Main from './Main';

describe ("Main", () => {
    it("without components", () => {
        const wrapper = shallow(<Main/>);
        expect(wrapper).toMatchSnapshot();
    })
})