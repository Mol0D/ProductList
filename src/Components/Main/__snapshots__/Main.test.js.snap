// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main without components 1`] = `ShallowWrapper {}`;
